deposits.md

```diff
@@ -82,7 +83,7 @@ follows:
   [aliased][address-aliasing] by the deposit feed contract).
 - `context.calldata` set to `data`
 - `context.gas` set to `gasLimit`
-- `context.value` set to `value`
+- `context.value` set to `sendValue`
```

Comment:
Could there be a need for app-devs to access the `depositedValue` amount in the context?